pkg/controller/operator/seed/reconciler.go

```diff
@@ -425,7 +425,18 @@ func (r *Reconciler) reconcileConfigMaps(ctx context.Context, cfg *operatorv1alp
 
 	var kubeSystemCreators []reconciling.NamedConfigMapCreatorGetter
 
-	if creator := kubermaticseed.RestoreS3SettingsConfigMapCreator(cfg); creator != nil {
+	// For backward compatibility check both sources for backup and restore configuration.
+	var backupRestore kubermaticv1.SeedBackupRestoreConfiguration
+	if cfg.Spec.SeedController.BackupRestore.Enabled {
+		backupRestore.Enabled = true
+		backupRestore.S3Endpoint = cfg.Spec.SeedController.BackupRestore.S3Endpoint
+		backupRestore.S3BucketName = cfg.Spec.SeedController.BackupRestore.S3BucketName
```

Comment:
Maybe I'm dense, but why does `backupRestore = cfg.Spec.SeedController.BackupRestore` not work?